langchain/src/vectorstores/prisma.ts

```diff
@@ -276,6 +294,18 @@ export class PrismaVectorStore<
     return results;
   }
 
+  buildSqlFilterStr(filter: PrismaSqlFilter) {
+    return Object.keys(filter)
+      .map((key) => {
+        const value = filter[key];
+        if (typeof value === 'number') {
+          return `${key} = ${value}`;
```

Comment:
This does seem to be prone to SQL injection, especially when dealing with `value`. Will look into it more as well to see if we can handle it more safely.